services/workers/serverless.yml

```diff
@@ -4,41 +4,25 @@ provider:
   name: aws
   runtime: python3.8
   versionFunctions: false
+  stage: ${opt:stage, 'local'}
   region: "${env:AWS_DEFAULT_REGION}"
-
-  iamRoleStatements:
-    - Effect: 'Allow'
-      Action:
-        - "secretsmanager:GetRandomPassword"
-        - "secretsmanager:CreateSecret"
-        - "secretsmanager:ListSecrets"
-      Resource: '*'
-    - Effect: 'Allow'
-      Action:
-        - "secretsmanager:*"
-      Resource: { Fn::ImportValue: '${self:custom.projectEnvName}-databaseSecretArn' }
+  stackName: ${file(./workers.conf.${self:provider.stage}.yml):stackName}
```

Comment:
I like the file approach, but we only need two files. One for all deployed envs, one for local